### PR TITLE
moved the package in notebooks from pyautogen to autogen

### PR DESCRIPTION
**Pull request by Hk669**
## Why are these changes needed?

to help the developers use the `autogen` package and not the `pyautogen`, i've only updated the notebooks and not the entire files. 

## Checks

- [ ] I've included any doc changes needed for https://autogen-ai.github.io/autogen/. See https://autogen-ai.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

File: notebook/JSON_mode_example.ipynb [hunk 1]
```diff
@@ -40,7 +40,7 @@
    "outputs": [],
    "source": [
     "%%capture --no-stderr\n",
-    "# %pip install \"pyautogen>=0.2.3\"\n",
+    "# %pip install \"autogen==0.3.0\"\n",
```
Comment (sonichi) [inline notebook/JSON_mode_example.ipynb]:
```suggestion
    "# %pip install \"autogen>=0.3.0\"\n",
```

File: notebook/agentchat_lmm_llava.ipynb [hunk 2]
```diff
@@ -26,7 +26,7 @@
    "source": [
     "### Before everything starts, install AutoGen with the `lmm` option\n",
     "```bash\n",
-    "pip install \"pyautogen[lmm]>=0.2.3\"\n",
+    "pip install \"autogen[lmm]>=0.2.3\"\n",
```
Comment (sonichi) [inline notebook/agentchat_lmm_llava.ipynb]:
```suggestion
    "pip install \"autogen[lmm]>=0.3.0\"\n",
```

File: notebook/agentchat_with_memory.ipynb [hunk 3]
```diff
@@ -17,7 +17,7 @@
    "source": [
     "This notebook demonstrates an intelligent customer service chatbot system that combines:\n",
     "\n",
-    "- PyAutoGen for conversational agents\n",
+    "- autogen for conversational agents\n",
```
Comment (sonichi) [inline notebook/agentchat_with_memory.ipynb]:
```suggestion
    "- AutoGen for conversational agents\n",
```

File: notebook/agentchat_lmm_llava.ipynb (hunk 2 above)
Comment (randombet) [inline notebook/agentchat_lmm_llava.ipynb]: version 0.3.0?

File: website/docs/Migration-Guide.md [hunk 4]
```diff
@@ -3,7 +3,7 @@
 ## Migrating to 0.2
 
 openai v1 is a total rewrite of the library with many breaking changes. For example, the inference requires instantiating a client, instead of using a global class method.
-Therefore, some changes are required for users of `pyautogen<0.2`.
+Therefore, some changes are required for users of `autogen<0.2`.
```
Comment (sonichi) [inline website/docs/Migration-Guide.md]:
```suggestion
Therefore, some changes are required for users of `pyautogen<0.2`.
```
This sentence is only relevant for pyautogen users.
We probably should add a section "migration guide to 0.3" and talk about the package name change and the future plan for retiring pyautogen.

File: notebook/agentchat_lmm_llava.ipynb (hunk 2 above)
Comment (Hk669) [reply notebook/agentchat_lmm_llava.ipynb]: thanks @randombet , this is resolved.

**Review by yiranwu0 — APPROVED** [review]

**Pull request merged**